webpack.config.js

```diff
@@ -15,10 +15,18 @@ module.exports = (env, args) => {
     },
     module: {
       rules: [
+        {
+          test: /\.tsx?$/,
+          loader: "awesome-typescript-loader"
```

Comment:
For a gradual transition, it might be a better idea to continue to use Babel for _both_ JS and TypeScript files, to prevent inconsistent output or bugs caused by differing implementations.

To do this, you could enable the TypeScript plugin for Babel, and run typechecking separately using the ForkTSChecker Webpack plugin. This should also make compilation faster, and absolve the need to run tests with ts-node :)

For inspiration, see [the PR on solid-panes](https://github.com/solid/solid-panes/pull/75/files?file-filters%5B%5D=.js&file-filters%5B%5D=dotfile) and [the one on solid-ui](https://github.com/solid/solid-ui/pull/127/files?file-filters%5B%5D=.json).

Reply:
Good points, will do!

Reply:
Done! Can you review again?

Reply:
Done, but not that I'm not an rdflib maintainer, so my review is more to provide feedback rather than a path towards merging...